### PR TITLE
Update shareable-configurations-list.md

### PR DESCRIPTION
**Pull request by jedmao**

### CODE REVIEW

File: docs/extending/shareable-configurations-list.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 - [@semantic-release/gitlab-config](https://github.com/semantic-release/gitlab-config) - semantic-release shareable configuration for GitLab
 
 ## Community configurations
+- [@jedmao/semantic-release-npm-github-config](https://github.com/semantic-release/semantic-release) - for publishing npm packages with [GitHub](https://github.com)
```
Comment (gr2m) [inline docs/extending/shareable-configurations-list.md]: How does that distinguish from semantic-release default setup, which also publishes to npm and github?

Comment (jedmao) [reply docs/extending/shareable-configurations-list.md]: ## Default

```json
    plugins: [
      '@semantic-release/commit-analyzer',
      '@semantic-release/release-notes-generator',
      '@semantic-release/npm',
      '@semantic-release/github',
    ],
```

## Mine

```json
	"plugins": [
		"@semantic-release/commit-analyzer",
		"@semantic-release/release-notes-generator",
		[
			"@semantic-release/changelog",
			{
				"changelogTitle": "# Changelog\n\nAll notable changes to this project will be documented in this file. See\n[Conventional Commits](https://conventionalcommits.org) for commit guidelines."
			}
		],
		[
			"@semantic-release/npm",
			{
				"tarballDir": "dist"
			}
		],
		[
			"@semantic-release/github",
			{
				"assets": "dist/*.tgz"
			}
		],
		[
			"@semantic-release/git",
			{
				"message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
			}
		]
	]
```

Comment (gr2m) [reply docs/extending/shareable-configurations-list.md]: Okay so this publishes to npm but also creates a tarball that gets uploaded to each GitHub release, plus it commits the version change in package.json and a CHANGELOG file? 

What I mean is your addition should explain in `shareable-configurations-list.md` how it differentiates from the defaults

Comment (jedmao) [reply docs/extending/shareable-configurations-list.md]: Did my best to summarize expectations and squash this PR down into 1 commit. Let me know if there's anything else I can clarify. Thanks!

**Pull request merged**